ClemBot.Bot/bot/cogs/translate_cog.py

```diff
@@ -146,75 +97,287 @@ async def translate_given_lang(self, ctx, input):
 
+LANGUAGE_NAME_TO_SHORT_CODE = {
+"Afrikaans":"af",      
+"Albanian":"sq",      
+"Amharic":"am",      
+"Arabic":"ar",      
+"Armenian":"hy",      
+"Assamese":"as",      
+"Azerbaijani":"az",      
+"Bangla":"bn",      
+"Bashkir":"ba",      
+"Bosnian (Latin)":"bs",      
+"Bulgarian":"bg",      
+"Cantonese (Traditional)":"yue",        
+"Catalan":"ca",      
+"Chinese (Literary)":"lzh",        
+"Chinese Simplified":"zh-Hans",          
+"Chinese Traditional":"zh-Hant",          
+"Croatian":"hr",      
+"Czech":"cs",      
+"Danish":"da",      
+"Dari":"prs",        
+"Divehi":"dv",      
+"Dutch":"nl",      
+"English":"en",      
+"Estonian":"et",      
+"Fijian":"fj",      
+"Filipino":"fil",        
+"Finnish":"fi",      
+"French":"fr",      
+"French (Canada)":"fr-ca",        
+"Georgian":"ka",      
+"German":"de",      
+"Greek":"el",      
+"Gujarati":"gu",      
+"Haitian Creole":"ht",      
+"Hebrew":"he",      
+"Hindi":"hi",      
+"Hmong Daw":"mww",        
+"Hungarian":"hu",      
+"Icelandic":"is",      
+"Indonesian":"id",      
+"Inuinnaqtun":"ikt",        
+"Inuktitut":"iu",      
+"Inuktitut (Latin)":"iu-Latn",          
+"Irish":"ga",      
+"Italian":"it",      
+"Japanese":"ja",      
+"Kannada":"kn",      
+"Kazakh":"kk",      
+"Khmer":"km",      
+"Klingon":"tlh-Lat",          
+"Klingon (plqaD)":"tlh-Piq",          
+"Korean":"ko",      
+"Kurdish (Central)  ":"ku",      
+"Kurdish (Northern)":"kmr",        
+"Kyrgyz":"ky",      
+"Lao":"lo",      
+"Latvian":"lv",      
+"Lithuanian":"lt",      
+"Macedonian":"mk",      
+"Malagasy":"mg",      
+"Malay ":"ms",      
+"Malayalam":"ml",      
+"Maltese":"mt",      
+"Maori":"mi",      
+"Marathi":"mr",      
+"Mongolian (Cyrillic)":"mn-Cyrl",          
+"Mongolian (Traditional)":"mn-Mong",          
+"Myanmar":"my",      
+"Nepali":"ne",      
+"Norwegian":"nb",      
+"Odia":"or",      
+"Pashto":"ps",      
+"Persian ":"fa",      
+"Polish":"pl",      
+"Portuguese (Brazil)":"pt",      
+"Portuguese (Portugal)":"pt-pt",        
+"Punjabi":"pa",      
+"Queretaro Otomi":"otq",        
+"Romanian":"ro",      
+"Russian":"ru",      
+"Samoan":"sm",      
+"Serbian (Cyrillic)":"sr-Cyrl",          
+"Serbian (Latin)":"sr-Latn",          
+"Slovak":"sk",      
+"Slovenian":"sl",      
+"Spanish":"es",      
+"Swahili":"sw",      
+"Swedish":"sv",      
+"Tahitian":"ty",      
+"Tamil":"ta",      
+"Tatar":"tt",      
+"Telugu":"te",      
+"Thai":"th",      
+"Tibetan":"bo",      
+"Tigrinya":"ti",      
+"Tongan":"to",      
+"Turkish":"tr",      
+"Turkmen":"tk",      
+"Ukrainian":"uk",      
+"Upper Sorbian":"hsb",        
+"Urdu":"ur",      
+"Uyghur":"ug",      
+"Uzbek (Latin)":"uz",      
+"Vietnamese":"vi",      
+"Welsh":"cy",      
+"Yucatec Maya":"yua",                           
+                         
+}
+CHUNK_SIZE = 15
+LANGUAGE_SHORT_CODE_TO_NAME = {value: key for key, value in LANGUAGE_NAME_TO_SHORT_CODE.items()}
+
+LOWERCASE_LANGUAGENAME = {
```

Comment:
This could be done using dictionary comprehension, see: https://www.datacamp.com/community/tutorials/python-dictionary-comprehension